docs/current_docs/manuals/user/cloud/741031-user-interface.mdx

```diff
@@ -3,24 +3,64 @@ slug: /manuals/user/cloud/741031/user-interface
 ---
 
 # User Interface
+:::info
+Dagger 0.11.0 adds support for visualizing Dagger Function calls in Dagger Cloud. With this update, Dagger Cloud will default to use the new **Traces** views for both `dagger call` and `dagger run` commands. Dagger v0.11.0 or later will automatically use the new **Traces** views. Older versions of Dagger will continue to use the legacy Runs experience.
+:::
 
-Dagger Cloud lets you visualize your Dagger pipelines in two ways: runs and changes.
-
-## Runs
+Dagger Cloud visualizes your Dagger pipelines runs invoked from your local dev machine and in CI. 
 
 A run is an invocation of a Dagger pipeline. When you run Dagger in CI, each call aligns with one run in Dagger Cloud. If you have many CI jobs configured to run for every pull/merge request, you may see several runs for a single commit.
 
 For instance, if 4 GitHub Actions workflows each call Dagger to run a pipeline for pull requessts, you will see four distinct runs in the Dagger Cloud UI. You may also see multiple runs if one GitHub workflow has multiple jobs defined with distinct calls to Dagger to run your pipelines. As you Daggerize your pipelines, you will likely have fewer distinct CI workflows.
 
 Because Dagger Cloud understands the Dagger engine's data model, it gives you visibility into the steps of your pipeline's Directed Acyclic Graph (DAG), including detailed logs for each step.
 
+## Traces
+:::info
+You will only see runs in the **Traces** view if you are using Dagger 0.11.0 or later. If you are using an older version of Dagger, you will see runs in the **Runs** view.
+:::
+
 :::note
-The Dagger team is working on a new user experience to group your Dagger runs by the pull or merge request (group commits by PR/MR).
+**Traces** is still under development and is in Beta. We welcome your feedback on how we can improve this page. Please join the [Dagger Discord](https://discord.gg/dagger-io) to join the Dagger community and share your feedback.
+:::
+
+### All Traces page
+The **Traces** page lists runs you've triggered from your local development host or in CI.
+
+![View Traces](/img/current_docs/manuals/user/cloud/user-interface/all-traces.png)
+
+Here's a quick summary of what you'll see for each trace.
+
+|Field | Description |
+|---|---|
+|Status icon | Indication of run success or failure |
```

Comment:
```suggestion
|Status icon | Indication of run processing, success or failure |
```
I don't know if `processing` is the best word. `run running` is redundant. Your call @d3rp3tt3

Reply:
I'll use "Indication of the runs status: in progress, succeeded, or failed"